### PR TITLE
Mentors popup for desktop

### LINKED ISSUE #136: Mentor popup DESKTOP
- [x] Implement a component for this mentor popup modal (only desktop for this sprint). You don't have to hook it up to the data yet. 
Screenshot 2023-01-21 at 4 08 49 PM

### PR DESCRIPTION
**Pull request by deanframe**
Adds a mentor popup for desktop

Fixes #136 .

Changelist:

- Clicking on a mentor in the list opens a popup
- Popup designed according to hifi
- Mostly hooked up to data- currently the contact button does nothing

Notes:

- Not responsive
- Some design decisions I want input on:
  - Currently I'm not dimming the background outside the modal - is that something we want?
  - Currently users can scroll the page while the modal is displayed, should that be changed?
- This PR also fixes a ton of console errors. The console should be clean now, except for gatsby/webpack/vercel notices.
- This PR also fixes some formatting issues 

Tested:

- locally firefox + chrome on windows 10

### CODE REVIEW

File: live-site/components/event-schedule-section/EventScheduleSection.styles.tsx [hunk 1]
```diff
@@ -38,9 +38,16 @@ const StyledEventScheduleSubText = styled(P)`
...
 
-const StyledBoldSubText = styled.p`
+const StyledBoldSubText = styled(P)`
+  color: ${colors.TEXT_BROWN};
   font-weight: bold;
   margin: 0;
+  @media ${max.tablet} {
+    text-align: left;
+    font-size: 1em;
+    margin-left: 1.5em;
```
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/EventScheduleSection.styles.tsx]: margin: 0 1.5em;

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
THIS LOOKS FABULOUS - AMAZING WORK DEAN!!!!

I was wondering for mentors that have a shift starting on Thursday but ending on another day (like if it was Thursday 10pm - Friday 2am) - what would this display? 

Currently in the database the shift for this mentor starts 1/19 at 10:59pm and ends 1/31 at 10:59pm but it just says Thursday 10:59pm - 10:59pm

I think there might be a better way to represent these cases? Like if it was 10pm - 2am shift to do Thursday 10pm - Friday 2am or something idk 
Screenshot 2023-01-26 at 10 39 49 AM

**Review by jzhzhang66 — APPROVED** [review]
LGTM

**Review by jhylisa666 — APPROVED** [review]
LGTMMM

**Pull request merged**